cpp/include/cudf/hashing/detail/hashing.hpp

```diff
@@ -82,7 +82,7 @@ std::unique_ptr<column> xxhash_64(table_view const& input,
  * @param rhs The second hash value
  * @return Combined hash value
  */
-constexpr uint32_t hash_combine(uint32_t lhs, uint32_t rhs)
+CUDF_HOST_DEVICE constexpr uint32_t hash_combine(uint32_t lhs, uint32_t rhs)
```

Comment:
I could be wrong but this function seems to be device-only as well.

Reply:
I see it [here](https://github.com/rapidsai/cudf/blob/branch-25.02/cpp/include/cudf/detail/aggregation/result_cache.hpp#L38) in host code.